src/components/views/donate/common.types.tsx

```diff
@@ -1,3 +1,10 @@
+import { ChainType } from '@/types/config';
+
 export interface ISwitchNetworkToast {
 	acceptedChains?: number[];
 }
+
+export interface INetworkIdWithChain {
+	chainType: ChainType;
+	networkId: number;
+}
```

Comment:
Just a suggestion to pass config easier.
```suggestion
export type INetworkIdWithChain = Pick<NetworkConfig, 'id' | 'chainType'>;
```
If we define NonEVMChain as below, we could pass them directly as well
```ts
export interface NonEVMChain
	extends Pick<NetworkConfig, 'id' | 'name' | 'chainType'> {
	adapterNetwork: WalletAdapterNetwork;
	blockExplorers: {
		default: {
			name: string;
			url: string;
		};
	};
}
```